sonarts-core/src/cfg/builder.ts

```diff
@@ -174,7 +180,9 @@ class CfgBuilder {
     }
     if (breakable) {
       const continueTarget = breakable.continueTarget;
-      return this.createBlockPredecessorOf(continueTarget!);
+      const continueBlock = this.createBlockPredecessorOf(continueTarget!);
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Forbidden non null assertion ([typescript:S2966](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS2966))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=sonarts&pullRequest=628&issues=AWNE6bCQlRUrfRWxk6O2&open=AWNE6bCQlRUrfRWxk6O2)

[ISSUE_KEY]: AWNE6bCQlRUrfRWxk6O2